### ISSUE TITLE
DatePicker value isn't reflected into the input form.

### ISSUE THREAD

**Issue by yumemi-omi** [element-plus/element-plus#609]
[Here](https://codesandbox.io/s/bold-brook-f8esz?fontsize=14&hidenavigation=1&theme=dark) is sample code.
Input value isn't reflected into the input form.

```
<template>
  <div class="block">
    <span class="demonstration">Default</span>
    <el-date-picker v-model="selectedDateRef" type="date"></el-date-picker>
  </div>
</template>


<script>
import { ref } from "vue";
export default {
  setup() {
    const selectedDateRef = ref("");

    return {
      selectedDateRef,
    };
  },
};
</script>

```

**Comment by yumemi-omi** [comment]
No response?

**Comment by JeremyWuuuuu** [comment]
Sorry kept you waiting, I am currently investigate this bug, hopefully I can make a fix for this bug.

**Comment by zazzaz** [comment]
This is might be related to this https://github.com/element-plus/element-plus/issues/603#issuecomment-730878604

**Comment by JeremyWuuuuu** [comment]
I have tried to reproduce this on my machine which is on MacOS Big Sur 11.0.1 (20B29)
And I am working on `Vue` with version 3.0.2.
![image](https://user-images.githubusercontent.com/15975785/99902744-0ce9e280-2cfb-11eb-82bd-1078f8a87e42.png)
Everything seems working fine on my machine, but somehow on the `CodeSandBox` link you provided it didn't.

**Comment by yumemi-omi** [comment]
Thank you for confirming it. 
I see, let me try again. And inform you of result later.

**Comment by JeremyWuuuuu** [comment]
According to #603, the root cause should be the compiler which compiles the code unexpectedly, we will make a work around for this issue

**Comment by JeremyWuuuuu** [comment]
Linking this to https://github.com/vuejs/vue-next/issues/2540

**Comment by JeremyWuuuuu** [comment]
This is considered fixed due to https://github.com/vuejs/vue-next/issues/2540 is merged, upgrade your local version of Vue to `> 3.0.5`.
